tools/sign.ps1

```diff
@@ -128,5 +129,20 @@ if ($LastExitCode) { Write-Error "signtool.exe exit code: $LastExitCode" }
 & $SignToolPath sign /f $CertPath -p "placeholder" /fd SHA256 $XdpFnLwfCat
 if ($LastExitCode) { Write-Error "signtool.exe exit code: $LastExitCode" }
 
+# Copy and sign the installer artifacts.
+$XdpInstallerInf = Join-Path $XdpMsiDir "xdp.inf"
+$XdpInstallerCat = Join-Path $XdpMsiDir "xdp.cat"
+$XdpInstallerSys = Join-Path $XdpMsiDir "xdp.sys"
+if (-not (Test-Path -Path $XdpMsiDir)) {
+    # Create the destination folder if it doesn't exist (installer not yet built).
+    New-Item -ItemType Directory -Path $XdpMsiDir | Out-Null
+}
+Copy-Item -Path "$RootDir\src\xdpinstaller\xdp.inf" -Destination $XdpInstallerInf
+Copy-Item -Path "$XdpDir\xdp.sys" -Destination $XdpInstallerSys
+& $Inf2CatToolPath /driver:$XdpMsiDir /os:10_x64
+if ($LastExitCode) { Write-Error "inf2cat.exe exit code: $LastExitCode" }
+& $SignToolPath sign /f $CertPath -p "placeholder" /fd SHA256 $XdpInstallerCat
+if ($LastExitCode) { Write-Error "signtool.exe exit code: $LastExitCode" }
```

Comment:
This seems to be copies of what is already generated above. Is that correct? If so, why can't those be used directly?

Reply:
They are not copies: the INF file is different, and consequently the generated CAT file is too (see PR's "TBD" note above).